Pocket/pocket-ios
# Fix tags not deleted from edit tags screen

## Goal
* In the "Edit Tags" screen accessible from the overflow menu of the Reader in a saved item, deleting a tag and tapping "Save" did not actually delete the tag. this PR fixes the bug.
* This PR also updates Sentry to `8.34.0`

## Test Steps
* Build/run this branch
* From either the Saves screen or the Recent Saves section on Home, select a saved item
* Tap the overflow menu, then "Edit Tags"
* Edit tags at will, and particularly delete existing ones, and make sure they are actually deleted
* Go back in the saves list, pull to refresh and make sure deleted tags do not reappear in the selected item.

## Screenshots
